docs/UserGuide.md

```diff
@@ -1083,38 +1088,108 @@ Read Book
+Then, all the pages of the section are represented as described in [6.2.1. Pages](#6.2.1.-Pages). 
 
-This is followed by `s` sets of lines, describing each of the `s` sections in notebook `i`. The `j`th set describes
-the `j`th section in the `i`th notebook.
+For example, a section might be represented like this: 
 
-The first line in the `j`th set contains the title of the `j`th section of the `i`th notebook.
+```
+Java
+2
+What is Java?
+Java is a programming language used by 3 billion devices.
+Why Java?
+It's highly portable and relatively fast.
+```
 
-The next line in the set contains only an integer number `p` indicating the number of pages in the `j`th section of
- the `i`th notebook.
+The first line, "Java", is the title of the section. 
 
-This is followed by `p` sets of 2 lines each. The `k`th set describes the `k`th page, of the `j`th section, of the `i
-`th notebook.
+The second line indicates that the section has 2 pages. 
 
-The first line of the `k`th set contains the title of the page.
+Then, we have 2 pages, titled "What is Java" and "Why Java" respectively. They are represented as described in [6.2.1. Pages](#6.2.1.-Pages). 
 
-The second line of the `k`th set contains the content of the page. The newline characters in the content of the
-page are replaced by `~~~`.  
+#### 6.2.3. Notebooks
 
-Here is an example of a notebooks save file:
+The first line of a notebook contains the title of the notebook. 
+
+The second line of a notebook contains just a number. This number indicates the number of sections in the notebook. 
+
+Then, all the sections of the notebook are represented as described in [6.2.2. Sections](#6.2.2.-Sections). 
+
+For example, a saved notebook might look like this: 
+
+```
+CS2113T
+4
+Java
+2
+What is Java?
+Java is a programming language used by 3 billion devices.
+Why Java?
+It's highly portable and relatively fast.
+OOP
+1
+What is OOP?
+OOP is Object Oriented Programming.
+UML
+2
+What is UML?
+UML is Unified Modeling Language.
+Why UML?
+UML is used to draw diagrams to explain your code to noobs.
+Assertions
+1
+Why assertions?
+Honestly I don't know.
+```
+
+The first line, "CS2113T", is the title of the notebook. 
+
+The second line indicates that the notebook has 4 sections. 
+
+Then, we have 4 sections, titled "Java", "OOP", "UML" and "Assertions" respectively. They are represented as described in [6.2.2. Sections](#6.2.2.-Sections). 
+
+#### 6.2.4. The Notebook Shelf
+
+The notebook shelf is what is stored in the save file. It represents all your notebooks, sections, and pages at once. It is formatted as follows. 
+
+The first line contains just a number. This number indicates the number of notebooks in your Notebook Shelf. 
+
+Then, all the notebooks on your Notebook Shelf are reprsesented as described in [6.2.3. Notebooks](#6.2.3.-Notebooks). 
+
+For example, a saved notebook shelf might look like this: 
```

Comment:
This section may seem a bit confusing? I think putting a screenshot here would be better.

Reply:
I'll see what I can come up with.